Add wayne_land_bank spider

radoslawkrolikowski opened the pull request:
This pull request adds the wayne_land_bank spider, the test file of that spider and target web page as an HTML file. 

-  To access the Board of Directors tab on the following page: https://public-wclb.epropertyplus.com/landmgmtpub/app/base/customPage we have to start with POST request.

- The desired data was included in `<script>` tags, thus to extract the data, the `re` regular expressions were used.

- The target site seems to be under the development, in the future the spider might require update if the website layer was changed or new information were added.

pjsier commented on city_scrapers/spiders/wayne_land_bank.py:
```diff
@@ -0,0 +1,76 @@
+
+        for item in data:
+            meeting = Meeting(
+                title=item[2].strip(" .()"),
```
The title should never be an empty string, if it's blank we should default to "Board of Directors". Since these are all for the Board, any additional title should probably be added to "Board of Directors " so that the special meeting becomes "Board of Directors Special Meeting"

pjsier commented on city_scrapers/spiders/wayne_land_bank.py:
```diff
@@ -0,0 +1,76 @@
+                end=None,
+                all_day=False,
+                time_notes="",
+                location=self._parse_location(""),
```
The `_parse_location` method here isn't necessary, but a `_validate_location` method would be good here to check if the location has changed. https://github.com/City-Bureau/city-scrapers/blob/c23f1fd17b1ced1c01dff295c8cdf1ca191f0244/city_scrapers/spiders/chi_ssa_34.py#L58-L60

pjsier commented on city_scrapers/spiders/wayne_land_bank.py:
```diff
@@ -0,0 +1,76 @@
+    start_urls = ["https://public-wclb.epropertyplus.com/landmgmtpub/app/base/customPage"]
+    location = {
+        "name": "Wayne County Treasurer's Office",
+        "address": "400 Monroe St 5th Floor, Detroit, MI 48226, United States",
```
"United States" isn't necessary here for our purposes

radoslawkrolikowski commented:
All changes that you suggested have been made. I am only wondering about the location item. I used default location name and address because it looks clearer than this one on the website. After adding `_validate_location` method, maybe it will be good to use scraped location if default wasn't found.
```
loc = re.findall('The Board of Directors holds meetings at (.*?)(?=\\n)', self.response_data[0])[0]
print('Name: {}, Address: {}'.format(loc.split('at')[0].title(), loc.split('at')[1]))
```
Or raising an error as it is right now is better option?

pjsier commented:
@radoslawkrolikowski thanks for these changes!  For now, since the markup is so irregular I think it's safest to throw the error if the location isn't found. Reviewing now but I think these are good to go
